doc/examples/anim13/anim13.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+#
+# Movie of seismic waveforms from 2020-10-06 Alaska earthquake
+# Epicenter 54.844N 159.842W
+# Depth 31.0 km
+# UTC time 2020-10-06 05:54:50
+# Recorded at station Dolgoi Island, Mount Dutton, Alaska
+#
+# USGS event page: https://earthquake.usgs.gov/earthquakes/eventpage/us6000c5zm/executive
+#
+# DEM:  @earth_relief_06m
+# Data: @waveform_AV.DOL.txt
```

Comment:
Need to explain the meaning of each column in the data `waveform_AV.DOL.txt`, as it's not a commonly used format in seismology community

Reply:
OK, might you be able to help so that it makes sense to other seismologists?